src/plugins/discover/public/embeddable/saved_search_embeddable.tsx

```diff
@@ -153,7 +153,7 @@ export class SavedSearchEmbeddable
         this.searchProps &&
         (titleChanged ||
           this.isFetchRequired(this.searchProps) ||
-          this.isInputChangedAndRerenderRequired(this.searchProps))
+          this.rerenderRequired(this.searchProps))
```

Comment:
Nit: I'd suggest going for 'isRenderRequired', for alignment with other functions starting with a verb... also it's simpler

Reply:
Done